android/samples/getuserscreenlet/local.properties

```diff
@@ -0,0 +1,12 @@
+## This file is automatically generated by Android Studio.
+# Do not modify this file -- YOUR CHANGES WILL BE ERASED!
+#
+# This file must *NOT* be checked into Version Control Systems,
+# as it contains information specific to your local configuration.
+#
+# Location of the SDK. This is only used by Gradle.
+# For customization when using a Version Control System, please read the
+# header note.
+#Fri Jun 03 14:19:31 CEST 2016
+sdk.dir=/Users/aelian/Library/Android/sdk
+bintray.user=nhpatt
```

Comment:
I saw I added it, but I didn't deleted it because it'll be recreated automatically and it didn't had secret information. I'll remove it.